docs/Install-with-Docker.md

```diff
@@ -58,45 +66,70 @@ docker --version
 docker-compose --version
 ```
 
-You should see a similar output:
+You should see a similar output to this;
 
 ![image](https://user-images.githubusercontent.com/75517/51273280-a132e200-19cc-11e9-914e-d54610f43ad6.png)
 
 **Note for Windows users**: you can use **git-bash** (the shell included in git) as a terminal.
 
+## PART 2 - AzerothCore
+
 ### Clone the AzerothCore repository
 
+Open a teminal such as Git Bash or Command Prompt as an Administrator.
```

Comment:
again, this is Windows-specific. On mac and linux it's just "open the terminal". I think it's better to specify in the beginning of the guide that for Windows users by "terminal" we mean "git bash".